package.json

```
@@ -1,76 +1,68 @@
 {
   "name": "jest-preset-stylelint",
   "version": "1.3.0",
-  "description": "Common configuration options for Jest in stylelint environments.",
+  "description": "Jest preset for stylelint plugins.",
   "keywords": [
     "stylelint",
     "jest",
     "jest-preset",
     "preset"
   ],
-  "author": "stylelint",
+  "repository": "stylelint/jest-preset-stylelint",
   "license": "MIT",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/stylelint/jest-preset-stylelint.git"
-  },
+  "author": "stylelint",
   "files": [
-    "create-jest-preset.js",
     "getOsEol.js",
+    "jest-preset.js",
     "jest-setup.js"
   ],
-  "engines": {
-    "node": ">=6"
-  },
-  "dependencies": {
-    "lodash": "^4.17.5",
-    "postcss-less": "^2.0.0"
-  },
-  "devDependencies": {
-    "eslint": "^5.1.0",
-    "eslint-config-stylelint": "^8.1.0",
-    "jest": "^23.4.1",
-    "jest-preset-stylelint": ".",
-    "npm-run-all": "^4.1.5",
-    "npmpub": "^4.0.1",
-    "remark-cli": "^5.0.0",
-    "remark-preset-lint-consistent": "^2.0.1",
-    "remark-preset-lint-recommended": "^3.0.1",
-    "stylelint": "^9.1.3"
-  },
-  "peerDependencies": {
-    "jest": ">=22",
-    "stylelint": "^10.1.0 || ^11.0.0 || ^12.0.0"
-  },
   "scripts": {
-    "dry-release": "npmpub --dry --verbose",
-    "lint:js": "eslint . --ignore-path .gitignore",
-    "lint:md": "remark . --quiet --frail",
+    "format": "prettier . --write",
     "lint": "npm-run-all --parallel lint:*",
-    "postinstall": "node create-jest-preset.js",
-    "pretest": "npm run lint",
-    "release": "npmpub",
+    "lint:formatting": "prettier . --check",
+    "lint:js": "eslint . ",
+    "lint:md": "remark . --quiet --frail ",
+    "release": "np",
     "test": "jest",
     "watch": "jest --watch"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": "eslint --cache --fix",
+    "*.{js,json,md,yml}": "prettier --write"
+  },
+  "prettier": "@stylelint/prettier-config",
   "eslintConfig": {
     "extends": [
       "stylelint"
-    ],
-    "parserOptions": {
-      "sourceType": "module"
-    },
-    "rules": {
-      "node/no-unpublished-require": "off"
-    }
-  },
-  "jest": {
-    "preset": "jest-preset-stylelint"
+    ]
   },
   "remarkConfig": {
     "plugins": [
-      "preset-lint-recommended",
-      "preset-lint-consistent"
+      "@stylelint/remark-preset"
     ]
+  },
+  "devDependencies": {
+    "@stylelint/prettier-config": "^2.0.0",
+    "@stylelint/remark-preset": "^1.0.0",
+    "eslint": "^6.8.0",
+    "eslint-config-stylelint": "^12.0.0",
+    "jest": "^25.3.0",
+    "np": "^6.2.1",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^2.0.4",
+    "remark-cli": "^8.0.0",
+    "stylelint": "^13.3.2"
+  },
+  "peerDependencies": {
+    "jest": ">=22",
```

```suggestion
    "jest": "^25.3.0",
```

Let's starts with the latest version, because maybe on `jest@25.2.0` it doesn't work. Also it's easier to support when we claim as little supported versions as possible :)